channels/stable-4.3.yaml

```diff
@@ -1,9 +1,27 @@
 name: stable-4.3
 versions:
-# 4.2.16+amd64 - withheld as it's only valid into 4.3.0,4.3.1 people should go direct to 4.3.5
+- 4.2.0
+- 4.2.1
+- 4.2.2
+# no 4.2.3 because it was rolled into 4.2.4
+- 4.2.4
+# no 4.2.5 because it was cut missing some intended update sources
+# no 4.2.6 because it pulled in CI-built images due to due to release job modification for multi-arch
+- 4.2.7
+- 4.2.8
+- 4.2.9
+- 4.2.10
+# - 4.2.11 failed to run tests, we never officially released it, but we accidentally
+# put it in a channel! (same for s390x)   Now we shouldn't pull it, just in case
+# someone is on it
+- 4.2.11
+- 4.2.12
+- 4.2.13
```

Comment:
When adding these to `candidate-4.3` (which [you should do in this PR][1]), you should also add a `blocked-edges/4.3.0-rc.0` to exclude `.*`, so we don't get a 4.2.13 -> 4.3.0-rc.0 in the candidate graph.  And a `blocked-edges/4.3.0-rc.3` to exclude `4\.2\.16`, because it's easy enough for those folks to flow down 4.2.z and jump off to 4.3 along a later, already stable, edge.  Edges into 4.3:

```console
$ for RC in 0 3; do oc adm release info "quay.io/openshift-release-dev/ocp-release:4.3.0-rc.${RC}-x86_64" | grep Upgrade; done
  Upgrades: 4.2.13
  Upgrades: 4.2.16, 4.3.0-rc.0, 4.3.0-rc.1, 4.3.0-rc.2
$ for z in $(seq 0 5); do oc adm release info "quay.io/openshift-release-dev/ocp-release:4.3.${z}-x86_64" | grep Upgrade; done
  Upgrades: 4.2.16, 4.3.0-rc.0, 4.3.0-rc.1, 4.3.0-rc.2, 4.3.0-rc.3
  Upgrades: 4.2.18, 4.3.0-rc.0, 4.3.0-rc.3, 4.3.0
  Upgrades: 4.2.19, 4.3.0, 4.3.1
  Upgrades: 4.2.20, 4.3.0, 4.3.1, 4.3.2
error: image does not exist
  Upgrades: 4.2.21, 4.2.22, 4.3.0, 4.3.1, 4.3.2, 4.3.3
```

So the other 4.2 sources are all in this stable-4.3 file already.

[1]: https://github.com/openshift/cincinnati-graph-data/pull/116/files#r392028195

Reply:
I'm not that concerned about candidate continuity but I think we should explore adding CI to ensure that `stable-4.x` is a subgraph of `fast-4.x` which is a subgraph of `candidate-4.x`.

Reply:
> I think we should explore adding CI to ensure that stable-4.x is a subgraph of fast-4.x which is a subgraph of candidate-4.x.

There's an open (internal) [ticket][1] for that.

[1]: https://issues.redhat.com/browse/OTA-126